modules/peddy/main.nf

```diff
@@ -15,7 +15,6 @@ process PEDDY {
     tuple val(meta), path("*.html")     , emit: html
     tuple val(meta), path("*.csv")      , emit: csv
     tuple val(meta), path("*.peddy.ped"), emit: ped
-    tuple val(meta), path("*.png")      , emit: png
```

Comment:
are these never generated?
If png files are only generated in some cases, the channel could become `optional`

Reply:
Agreed and implemented